ubi9-py39/Dockerfile

```diff
@@ -0,0 +1,51 @@
+# Thoth's extension to OpenShift's S2I build
+FROM registry.access.redhat.com/ubi9/python-39:1-59
```

Comment:
could you use registry.access.redhat.com/ubi9/python-39@sha256:6329d5779e89630928d5fa65a7b7b824f6bf41357f5db1688f0efafcbe48b14d